test/html/__snapshots__/normalize.test.js.snap

```diff
@@ -378,6 +308,6 @@ Object {
 
 exports[`html normalizer unclosed html <html><meta><link><img><br><hr><input></html> 1`] = `
 Object {
-  "actual": "<html xmlns=\\"http://www.w3.org/1999/xhtml\\"><meta/><link/><img/><br/><hr/><input/></html>",
+  "actual": "<html xmlns=\\"http://www.w3.org/1999/xhtml\\"><meta><link><img><br><hr><input/></hr></br></img></link></meta></html>",
```

Comment:
This is the most drastic/massive change that I might need to find a fix for before releasing 0.9.0, but to prevent this PR from growing indefinitely, it will have to be part of a later one.
Correct XML parsing has to be more important than the more lax/flexible HTML rules.

Reply:
Solved in [`275693d` (#519)](https://github.com/xmldom/xmldom/pull/519/commits/275693d6727113c100b42fd4e7f2d345ff854a15)